lib/spack/spack/cmd/style.py

```diff
@@ -252,20 +263,41 @@ def run_mypy(file_list, args):
 
     rewrite_and_print_output(output, args)
 
-    if returncode == 0:
-        tty.msg("mypy checks were clean")
-    else:
-        tty.error("mypy checks found errors")
+    print_tool_result("mypy", returncode)
+    return returncode
+
+
+def run_isort(file_list, args):
+    isort_cmd = which("isort")
+    if isort_cmd is None:
+        tty.error("isort is not available in path, skipping")
+        return 1
+
+    print_tool_header("isort")
+    check_fix_args = () if args.fix else ("--check", "--diff")
+
+    pat = re.compile("ERROR: (.*) Imports are incorrectly sorted and/or formatted")
+    replacement = "ERROR: {0} Imports are incorrectly sorted and/or formatted"
+    returncode = 0
+    for chunk in grouper(file_list, 100):
+        packed_args = check_fix_args + tuple(chunk)
+        output = isort_cmd(*packed_args, fail_on_error=False, output=str, error=str)
+        returncode |= isort_cmd.returncode
+
+        rewrite_and_print_output(output, args, pat, replacement)
+
+    print_tool_result("isort", returncode)
     return returncode
 
 
 def run_black(file_list, args):
     black_cmd = which("black")
     if black_cmd is None:
-        tty.error("style: black is not available in path, skipping")
+        tty.error("black is not available in path, skipping")
         return 1
 
     print_tool_header("black")
+    check_fix_args = () if args.fix else ("--check", "--diff", "--color")
```

Comment:
`--color` could be put in the `pyproject.toml`

Reply:
it doesn't matter because it'll still decide to decolorize it when we read the output.

Reply:
Color output works for me in GitHub Actions.